Kana module refactoring and extracting of Address module

vkublytskyi opened the pull request:
Kana module responsible only for handling kana fields.
Address module responsible for forms modification and address formatting.

vkublytskyi commented on Address/etc/config.xml:
```diff
@@ -0,0 +1,44 @@
+                <html><![CDATA[{{depend postcode}}〒{{var postcode}}{{/depend}}<br />
+{{depend region}}{{var region}}{{/depend}}{{depend city}}{{var city}}{{/depend}}{{depend street1}}{{var street1}} {{/depend}}{{depend street2}}{{var street2}}{{/depend}}{{depend street3}}{{var street3}}{{/depend}}<br />
+{{depend prefix}}{{var prefix}} {{/depend}}{{var lastname}} {{depend middlename}}{{var middlename}} {{/depend}}{{var firstname}}{{depend suffix}} {{var suffix}}{{/depend}}{{depend lastnamekana}} ({{var lastnamekana}} {{var firstnamekana}}){{/depend}}
+{{depend telephone}}<br />T: {{var telephone}}{{/depend}}
```
@HirokazuNishi, what characters or word we may use for Japan site instead of `T`, `F` . and `VAT`?

HirokazuNishi replied:
There is no VAT number in our country.
T. and F. are... depends on website owners, but I think T and F are OK.

vkublytskyi replied:
Thank you. I will remove VAT from default address templates and a merchant will be able to change them on Japanese Localization tab at Admin panel

Commit:
Kana module refactoring and extracting of Address module.

Kana module responsible only for handling kana fields.
Address module responsible for forms modification and address formatting.

Commit:
Fixed minor code issues.

Commit:
Removed VAT from default address template
